src/utils/wallet/providers/okx_wallet.ts

```diff
@@ -97,21 +107,21 @@ export class OKXWallet extends WalletProvider {
       throw new Error("OKX Wallet not connected");
     }
     // sign the PSBTs
-    return await window[okxProvider]?.bitcoinSignet?.signPsbts(psbtsHexes);
+    return await this.bitcoinNetwork.signPsbts(psbtsHexes);
   };
 
   signMessageBIP322 = async (message: string): Promise<string> => {
     if (!this.okxWalletInfo) {
       throw new Error("OKX Wallet not connected");
     }
-    return await window[okxProvider].bitcoinSignet?.signMessage(
+    return await this.bitcoinNetwork?.signMessage(
       message,
       "bip322-simple",
     );
   };
 
   getNetwork = async (): Promise<Network> => {
-    return "testnet";
+    return Network.TESTNET;
```

Comment:
```suggestion
    return Network.SIGNET;
```

Let's make sure that we don't have any inconsistencies between signet and testnet. We should use signet everywhere for now.